content/tutorials/manage-the-rippled-server/installation/rippled-1-3-0-migration-instructions.md

```diff
@@ -0,0 +1,74 @@
+# rippled v1.3.0 Migration Instructions
+
+This document describes the migration process for upgrading from `rippled` 1.2.4 or earlier to `rippled` v1.3.0 or later. This is necessary because the `rippled` install process has changed as of version 1.3.0.
+
+This document provides migration steps for upgrading on supported platforms:
+
+- [CentOS or Red Hat Enterprise Linux (RHEL)](#migration-on-centos-or-red-hat-enterprise-linux-rhel)
+- [Ubuntu Linux](#migration-on-ubuntu-linux)
+
+For other platforms, see the updated instructions for compiling from source. ([Ubuntu](build-run-rippled-ubuntu.html), [macOS](build-run-rippled-macos.html), or [Windows](https://github.com/ripple/rippled/tree/develop/Builds/VisualStudio2017))
+
+
+## Migration on CentOS or Red Hat Enterprise Linux (RHEL)
+
+Ripple's official RPM repository and instructions for using it have changed. To migrate from the old repository to the new one, complete the following steps:
+
+1. Stop the `rippled` server.
+
+        $ sudo systemctl stop rippled.service
+
+2. Replace Ripple's old yum repo file with the new one:
+
+        $ cat << REPOFILE | sudo tee /etc/yum.repos.d/ripple.repo
+        [ripple-stable]
+        name=XRP Ledger Packages
+        baseurl=https://repos.ripple.com/repos/rippled-rpm/stable/
+        enabled=1
+        gpgcheck=0
+        gpgkey=https://repos.ripple.com/repos/rippled-rpm/stable/repodata/repomd.xml.key
+        repo_gpgcheck=1
+        REPOFILE
+
+3. Fetch the latest repo updates:
+
+        $ sudo yum -y update
+
+4. Install the new `rippled` package:
+
+        $ sudo yum install rippled
+
+    Version 1.3.0 does not require any changes to your configuration files (`rippled.cfg` and `validators.txt`). This update procedure leaves your existing config files in place.
```

Comment:
Small nit, but you've been referring to configuration files everywhere else as "config files"

Reply:
Good point. Turns out we had ~14 instances of "configuration file" and ~74 instances of "config file". The latter is shorter and, although it's less formal, is pretty ubiquitous overall (outside our docs) so I've gone and replaced all instances of "configuration file" with "config file" for consistency.